packages/http-server/src/server.ts

```diff
@@ -80,23 +71,44 @@ export const createServer = (operations: IHttpOperation[], opts: IPrismHttpServe
 
       const { output } = response;
 
-      reply.code(output.statusCode);
-
-      if (output.headers) {
-        reply.headers(output.headers);
+      if (response.validations.output.length > 0) {
+        const violations = response.validations.output.map(detail => ({
```

Comment:
Is `violations` supported by any type? How about creating one + a builder which transforms `IDiagnostic` into that type.

Reply:
Hmm can you elaborate more? I am not sure I understand what you mean

Reply:
Yes, `violations` is currently a dynamic object. I'd be nice to have a type for it: `IHeaderDiagnostic` or sth similar.

Additionally, there could be a builder:
`const violations = response.validations.output.map(fromPrismDiagnostic)` (more or less)

Reply:
Ah ok ok I understand it now. Will see what I can do about it, sure!